### DISCUSSION TITLE
GitHub Overbilling a Suspended Account. Support Unresponsive and I am unable to cancel billing.

### DISCUSSION THREAD

**Discussion by amuelrath** [community/community#118124] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

A couple months back, as a new member of GitHub, I had my account suspended for what I later learned was 'overuse of github copilot' which was puzzling to me that I could be banned for using a paid tool without any explicit hard caps. 

Slow support wait times and an urgent need for use of copilot caused me to create a new account to use copilot on. I learned the hard way that this is against the TOS and that account was suspended as well.

This issue was later resolved and the original account was reinstated and the second account remained suspended. I recently learned after reviewing my billing statements that I have still been getting charged for copilot on the still-suspended account. I am unable to login and cancel and a support ticket I filed has been unanswered for (as of writing this) five days.

What else can be done? Are there any other avenues to stop this billing? This is starting to feel like theft.

Thanks.

**Comment by slutifsh99** [comment]
Hey there, I’m dealing with a similar issue. Could you share how you resolved it?
